package.json

```diff
@@ -1,118 +1,166 @@
 {
   "private": true,
+  "keywords": [],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/hicommonwealth/commonwealth.git"
+  },
+  "workspaces": [
+    "packages/commonwealth",
+    "packages/discord-bot",
+    "packages/snapshot-listener",
+    "libs/*"
+  ],
   "scripts": {
     "build": "chmod u+x scripts/build.sh && ./scripts/build.sh",
     "build-ci": "yarn global add node-gyp && yarn --ignore-engines && yarn build && yarn workspace commonwealth migrate-db",
-    "start": "concurrently -n watcher,server -c red,green 'tsc -b -w packages/commonwealth' 'yarn workspace commonwealth start'",
+    "chromatic": "chromatic --exit-zero-on-changes",
     "clean": "find . -name node_modules -exec rm -rf {} \\;; yarn --ignore-engines",
-    "heroku-prebuild": "yarn global add node-gyp",
-    "heroku-postbuild": "chmod u+x scripts/heroku-build.sh && ./scripts/heroku-build.sh",
-    "psql": "yarn --cwd packages/commonwealth psql",
-    "test-api": "yarn --cwd packages/commonwealth test-api",
     "dump-db": "yarn --cwd packages/commonwealth dump-db",
```

Comment:
Why are we moving all the database commands away from each other? I think for scripts it's cleaner to have similar functionality grouped rather than alphabetical ordering.

Reply:
The convention is to have package.json sorted for easy lookup... that's why scripts should start with the subject-verb, instead of verb-subject... we'll have a script convention soon to define these rules + dev/prod modes.

In the meantime we can leave it sorted I thought.